spec/lib/tasks/send_mail.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'rake_helper'
+
+RSpec.describe 'Tasks::SendMail' do
+  subject(:task) { Rake.application['send_mail:notice_liked_ranking'] }
+
+  let!(:hakjae) { create(:user, name: 'hakjae', email: 'hakjae@example.com') }
+  let!(:taji) { create(:user, name: 'taji', email: 'taji@example.com') }
+  let!(:uekkiesan) { create(:user, name: 'uekkiesan', email: 'uekkiesan@example.com') }
+
+  it '前日のいいね数トップ10の記事を全ユーザにメール送信できること' do
+    liked_sentences = %w[よろしく おはよう じゃあね 女の嫌は嫌じゃない スマイル マック食べる おやすみ 時計 レールカーテン お疲れ]
```

Comment:
（細かい）あくまでpostのセンテンスかなとおもった
```suggestion
    liked_post_sentences = %w[よろしく おはよう じゃあね 女の嫌は嫌じゃない スマイル マック食べる おやすみ 時計 レールカーテン お疲れ]
```